api/organisations/tasks.py

```diff
@@ -38,6 +38,17 @@ def send_org_over_limit_alert(organisation_id):
     )
 
 
+@register_task_handler()
+def send_org_subscription_cancelled_alert(
```

Comment:
I do understand this is a tiny method, but can we add a test for this as well?

Reply:
Haha, sure. Added one.